projects/quml-demo-app/src/app/quml-library-data.ts

```diff
@@ -43,6 +43,16 @@ export const data1 = {
         }
     },
     'metadata': {
+        "questionIds": [1,2,3,4],
+        "threshold" : 2,
```

Comment:
Move threshold to context object and remove question ids and get them from children, don't modify the metadata object